### ISSUE TITLE
Upgrading from 0.14 to 0.16 migration gets problem being detected

### ISSUE THREAD

**Issue by arthurzenika** [movim/movim#905]
Migrating from 0.14 (debian package) to source install on 0.16.1 : 

Starting says : 

```
movim-0.16.1$ php daemon.php start [snip]
The database needs to be migrated before running the daemon
To migrate the database run
php vendor/bin/phinx migrate
```

Then 

```
movim-0.16.1$ php vendor/bin/phinx migrate
Phinx by CakePHP - https://phinx.org.

using config file ./phinx.php
using config parser php
using migration paths 
 - /var/www/movim-0.16.1/database/migrations
using seed paths 
 - /var/www/movim-0.16.1/database/seeds
warning no environment specified, defaulting to: movim
using adapter pgsql
using database movim
ordering by creation time

All Done. Took 0.0584s
```

Then the message above keeps repeating.

**Comment by arthurzenika** [comment]
For lack of better solution, commenting the following line got the server working https://github.com/movim/movim/blob/master/src/Movim/Console/DaemonCommand.php#L70

**Comment by edhelas** [comment]
Yes this has been fixed in master and will be released with the 0.16.2 :)
